config/deploy/production.rb

```diff
@@ -1,6 +1,6 @@
 config = Net::SSH::Config.for 'incidents.pvta.com'
-remote_user = config[:user] || ENV[:user]
-port = config[:port] || ENV[:port]
+remote_user = config[:user] || ENV['USER']
+port = config[:port] || ENV['PORT']
```

Comment:
Note that this could end up being `nil`. But it looks like Capistrano sensibly takes `port: nil` to mean `port: 22`.

Reply:
OK